src/vision.cs

```diff
@@ -3122,6 +3445,16 @@ interface VNDetectHumanBodyPoseRequest {
 		[Static]
 		[Export ("currentRevision")]
 		VNDetectHumanBodyPoseRequestRevision CurrentRevision { get; }
+
+		[TV(17, 0), Mac(14, 0), iOS(17, 0), MacCatalyst(17,0)]
+		[Export("supportedJointsGroupNamesAndReturnError:")]
+		[return: NullAllowed]
+		string[] GetSupportedJointsGroupNames([NullAllowed] out NSError error);
```

Comment:
Objc returns a string, but is in reality an enum form the one you created: VNAnimalBodyPoseObservationJointsGroupName

Reply:
discussed offline with @mandel-macaque and decided to follow what was already being done in the framework and use the BindAs attribute to improve the exposed api